_posts/ref/2022-01-18-wire-graph-service.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: page
+title:  "WireGraphService Configuration Format"
+categories: [ref]
+---
+
+This document describes the format of the configuration of the WireGraphService component. The configuration of the WireGraphService contains the informations related to the Wire Graph topology and rendering properties.
+The pid of the WireGraphService configuration is `org.eclipse.kura.wire.graph.WireGraphService`.
+The WireGraphService configuration contains a single property of string type named `WireGraph` that contains a serialized JSON representation of a [WireGraph](#wire_graph) object describing the current graph layout.
```

Comment:
> The WireGraphService configuration contains a single property of string type named `WireGraph` that contains a serialized JSON representation of a [WireGraph](#wire_graph) object describing the current graph layout.

The WireGraphService configuration represents the current graph layout as a single string typed property named `WireGraph` that represents a serialized JSON representation of a [WireGraph](#wire_graph) object.